src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -90,9 +91,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     String replacement = "<img src=\"$1\" />";
 
+    String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
+    float sentimentScore = getSentimentScore(text);
+    
     String textWithImagesReplaced = userText.replaceAll(regex, replacement);
-
-    Message message = new Message(user, textWithImagesReplaced);
+    
+    Message message = new Message(user, text,textWithImagesReplaced);
```

Comment:
This doesn't work. The constructor takes user, text, and sentimentScore, not text and another text.

Reply:
ok fixed it!